controlplane/pkg/nsm/nsm_heal_processor_test.go

```diff
@@ -0,0 +1,488 @@
+package nsm
+
+import (
+	"context"
+	"fmt"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
+	local_connection "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/connection"
+	local_networkservice "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/networkservice"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/nsm"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/registry"
+	remote_connection "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/remote/connection"
+	remote_networkservice "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/remote/networkservice"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/model"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/nsmd"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/serviceregistry"
+	test_utils "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/tests/utils"
+	. "github.com/onsi/gomega"
+	net_context "golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"testing"
+	"time"
+)
+
+const (
+	networkServiceName = "golden_network"
+
+	localNSMName  = "nsm-local"
+	remoteNSMName = "nsm-remote"
+
+	dataplane1Name = "dataplane-1"
+	dataplane2Name = "dataplane-2"
+
+	nse1Name = "nse-1"
+	nse2Name = "nse-2"
+)
+
+var m model.Model
```

Comment:
Let's wrap it with struct and do not use global scope.

Reply:
Here it is : 5f1628c